catalogs/catalog-lakehouse-iceberg/src/main/java/com/datastrato/gravitino/catalog/lakehouse/iceberg/IcebergCatalogOperations.java

```diff
@@ -89,6 +89,7 @@ public void initialize(
 
     Map<String, String> resultConf = Maps.newHashMap(prefixMap);
     resultConf.putAll(gravitinoConfig);
+    resultConf.put("catalog_uuid", info.id().toString());
```

Comment:
Why do we need this?

Reply:
It was used to create the name of the key table that belongs to this catalog.

Reply:
Is it used for Iceberg library? Why don't we need it before?

Reply:
Other catalogs use `info.id()` instead, because Iceberg will initialize the corresponding backend in another method and If we would not like to change the method signature, we need to put the `uuid` in configuration.

Reply:
Could you elaborate more?

Reply:
Let me show you the code:
For Hive catalog
![image](https://github.com/datastrato/gravitino/assets/15794564/a765eabf-110d-4f9d-b6eb-754729de8f3e)
<img width="1066" alt="image" src="https://github.com/datastrato/gravitino/assets/15794564/19453b4f-2a87-47bc-a004-297ed985f015">

`info` is a field of `HiveCatalogOperataions`.


For Iceberg Hive backend

<img width="1164" alt="image" src="https://github.com/datastrato/gravitino/assets/15794564/f3ad4250-97b1-48b4-876a-e24e4dc3f1cc">

![image](https://github.com/datastrato/gravitino/assets/15794564/2ade906e-6f6c-47d3-b741-c97c59dc5af3)

I have to pass the id to the parameter here.

Reply:
I got it.